RoT/Protocol/RFCs/0003-RFC_PFM_Supported_Versions_Extension.md

```diff
@@ -0,0 +1,60 @@
+* Name: Extension for the PFM Supported Versions Command
+* Date: 2021-05-11
+* Pull Request: [#18](https://github.com/opencomputeproject/Security/pull/18)
+
+# Objective
+
+In the original design, there was only a single firmware component supported in
+a PFM.  The command to get the list of supported versions in a PFM didn't require
+any additional granularity.  With the introduction of version 2 of the PFM
+structure, there can now be multiple independent firmware components, each with
+their own list of supported firmware versions.  It is not always useful to get
+the full list of all versions for all components within a PFM.  Returning
+supported versions in this way would make it more challenging to search for a
+specific version of a specific component.  There needs to be a way to target a
+specific component's supported versions.
+
+# Proposal
+
+Update the Get Platform Firmware Manifest Supported Firmware command to have
+optional fields added to the request the specify the firmware component ID to
+query.  If these optional fields are not provided, every version for every
+component will be returned.
+
+# Specification Changelist
+
+The request structure in section 6.24 will be updated to include:
```

Comment:
The markdown document in this repo doesn't have section numbers, maybe link to it instead?

https://github.com/opencomputeproject/Security/blob/master/RoT/Protocol/Challenge_Protocol.md#get-platform-firmware-manifest-supported-firmware

Reply:
I removed the section number and added the command name and link instead.